.github/workflows/build_tag_push_image.yml

```diff
@@ -0,0 +1,53 @@
+name: When a pull request is merged to main, build the container, tag and push to the docker hub.
+
+on:
+  pull_request:
```

Comment:
Do you think this should happen on push to the main branch too? We haven't blocked that yet, I don't think...

Reply:
I can add it, but better to block direct commits to `main`.